src/tools/imgur/imguruploader.cpp

```diff
@@ -48,6 +52,13 @@ ImgurUploader::ImgurUploader(const QPixmap& capture, QWidget* parent)
     setWindowTitle(tr("Upload to Imgur"));
     setWindowIcon(QIcon(":img/app/flameshot.svg"));
 
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 10, 0))
```

Comment:
Do you know if there is an alternative API that supports QT 5.9?

Reply:
I am not sure which part of this is not supported in 5.9 :D

Reply:
Why is there a QT version check macro?

Reply:
ah, it's `screenAt` https://github.com/flameshot-org/flameshot/pull/318#issue-209315899